Source/Public/New-BicepParameterFile.ps1

```diff
@@ -11,7 +11,10 @@ function New-BicepParameterFile {
 
         [Parameter(Position = 3)]
         [ValidateNotNullOrEmpty()]
-        [string]$OutputDirectory
+        [string]$OutputDirectory,
+
+        [switch]
```

Comment:
Please use parameter sets, `-AsHashtable` should not be allowed to use with `-OutputDirectory`. Good names for parameter sets are `Default` and `AsHashtable, that would align with `Build-Bicep`.